services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -481,6 +485,7 @@ private void doInstallSwitchRules(final CommandMessage message, String replyToTo
             producerService.sendMessageAndTrack(replyToTopic, infoMessage);
 
         } catch (SwitchOperationException e) {
+            logger.error(format("Failed to install rules on switch %s", request.getSwitchId()), e);
```

Comment:
No need to use String.format when logging an exception as the last argument. See https://www.slf4j.org/faq.html#paramException

Reply:
good point. fixed